auth/server/src/main/java/com/seagates3/controller/PolicyController.java

```diff
@@ -64,6 +64,7 @@ public ServerResponse create() {
             policy = policyDAO.find(requestor.getAccount(),
                     requestBody.get("PolicyName"));
         } catch (DataAccessException ex) {
+          LOGGER.error("Failed to create policy");
```

Comment:
Pls log policy name along with error message.
General guideline, always log the message with context of error message with variables and values

Reply:
Please add additional info in log message at all places in this PR

Reply:
Fixed